packages/editor/src/components/document-outline/index.js

```diff
@@ -70,9 +70,6 @@ export const DocumentOutline = ( { blocks = [], title, onSelect, isTitleSupporte
 
 	let prevHeadingLevel = 1;
 
-	// Select the corresponding block in the main editor
-	// when clicking on a heading item from the list.
-	const onSelectHeading = ( clientId ) => onSelect( clientId );
 	const focusTitle = () => {
```

Comment:
This and all references to it would become dead code.

Reply:
removed